src/sdl2_compat.c

```diff
@@ -2559,37 +2559,11 @@ GestureAddTouch(const SDL_TouchID touchId)
     return &GestureTouches[GestureNumTouches++];
 }
 
-static int
-GestureDelTouch(const SDL_TouchID touchId)
```

Comment:
When was this function called in SDL2? I thought this was dynamically removing touches?

Reply:
Was called from within `SDL_DelTouch` in SDL_touch.c. Nowhere to call in here.